### PR TITLE
Alternate BField

### PR DESCRIPTION
**Pull request by brownd1978**
allow alternate bfield to be specified in primary generation and digitization.

### CODE REVIEW

**Comment by brownd1978** [comment]
Note that we need to add something to the output collection configuration field to specify the non-standard bfield.  I will update after your feedback.

**Comment by brownd1978** [comment]
@FNALbuild run build test with Mu2e/Offline#828

**Review by sophiemiddleton — APPROVED** [review]

File: Scripts/gen_Digitize.sh [hunk 1]
```diff
@@ -18,7 +19,11 @@ digitype=$6
 digout=${primary}${digitype}
 dbpurpose=$2_$7
 dbver=$8
-#
+#bfield="Offline/Mu2eG4/geom/bfgeom_no_tsu_ps_v01.txt"
+if [[ $# -eq 9 ]]; then
+  bfield="Offline/Mu2eG4/geom/$9"
```
Comment (rlcee) [inline Scripts/gen_Digitize.sh:24]: Might be a little more standard to not assume which directory the file is in and require the full path relative to the standard data path.  If you choose to not make this change, the relevant comment at the top of the file could make this choice more clear.

File: Scripts/gen_Digitize.sh [hunk 2]
```diff
@@ -62,7 +67,9 @@ else
 fi
 echo "Generating digitization scripts for $primary conf $conf output $digout database purpose, version $dbpurpose, $dbver"
 
-rm digitize.fcl
+if [[ -f "digitize.fcl" ]]; then
```
Comment (rlcee) [inline Scripts/gen_Digitize.sh:70]: I think you would get the same result with one line
`rm -f digitize.fcl`

**Review by rlcee — APPROVED** [review]
I included two small comments, up to you to decide what to do, approving now

File: Scripts/gen_Digitize.sh (hunk 1 above)
Comment (brownd1978) [reply Scripts/gen_Digitize.sh:24]: I'd prefer to leave this.  We've adopted that directory as the official source of BField maps, someone can add whatever maps they want to their clone of Offline to test, so it's not a restriction.  The command line for gen_*.sh are already too long/complicated.

File: Scripts/gen_Digitize.sh (hunk 2 above)
Comment (brownd1978) [reply Scripts/gen_Digitize.sh]: good point.  I updated this script and all the rest for the same issue.

**Comment by kutschke** [comment]
CI was run earlier this evening when Offline PR #828 was tested in combination with this one.  Merging now.

**Pull request merged**